rabbit-core/src/main/java/org/ohdsi/utilities/files/QuickAndDirtyXlsxReader.java

```diff
@@ -590,6 +592,7 @@ public String getStringByHeaderName(String fieldName) {
 		public Double getDoubleByHeaderName(String fieldName) {
 			String value = getStringByHeaderName(fieldName);
 			if (value != null) {
+				value = value.replace("<=","").trim();

```

Comment:
Is this the only operator you'd like to remove? Otherwise, you could also create a regex and do a replaceAll.
```java
// Ignore operators and spaces from double values
private static final Pattern DOUBLE_IGNORE_PATTERN = Pattern.compile("[<>= ]+");
...
value = DOUBLE_IGNORE_PATTERN.matcher(value).replaceAll("");
```